nix/cardano-services/deployments/coingecko-proxy-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
```

Comment:
As mentioned in my previous review this service should not exist here

Reply:
I agree with this. It's not a Cardano service, so best to remove and introduce it separately when the issue of it's home is determined. It's not being used by Lace yet so is not required to fulfil the needs of Lace developers.

Reply:
Is this a straight copy from wallet-world?

Reply:
Looks that way. I'm going to retract my change request in then interest of getting this initial setup complete. Any further refinements can be done separately.

Reply:
> Is this a straight copy from wallet-world?

Yes it is. :+1:

Reply:
yes it is, I think coingecko should be a separate deployment. it does not have much to do with `cardano-services` and it definitely does not belong in this repo

Also, this chart is  a little useless since it has image hardcoded thus no upgrade possible without manual intervention